src/transformers/models/auto/configuration_auto.py

```diff
@@ -26,6 +26,8 @@
 
 logger = logging.get_logger(__name__)
 
+SPECIAL_MODEL_TYPES_TO_CANONICAL_MODEL_TYPES = {"nllb": "m2m_100", "decision_transformer_gpt2": "decision_transformer"}
```

Comment:
In order to get the configuration classes, as these model types are not in `CONFIG_MAPPING_NAMES`